table.go

```diff
@@ -624,12 +624,13 @@ func (t *Table) printRow(columns [][]string, rowIdx int) {
 	}
 	//fmt.Println(max, "\n")
 	for x := 0; x < max; x++ {
+		var buffer strings.Builder
```

Comment:
Must it be `string.Builder` ?  the `&buffer` in every line does not make this clean